Doc/library/argparse.rst

```diff
@@ -1122,6 +1122,9 @@ is used when no command-line argument was present::
    >>> parser.parse_args([])
    Namespace(foo=42)
 
+For required_ arguments, the ``default`` value is ignored. For example, this
```

Comment:
Aside: we have a (somewhat unusual) convention of using italics for arguments instead of literal code formatting:

https://devguide.python.org/documentation/markup/

But I see we're using literals elsewhere in this file so it's best to be consistent.